styles/components/MyAccount.module.scss

```diff
@@ -72,6 +72,27 @@
     border-color: var(--nypl-colors-ui-gray-light-cool);
   }
   th:first-of-type {
-    width: 413px;
+    width: 427px;
   }
+  th {
+    width: 213px;
+  }
+}
+
+.statusBadge {
+  background: var(--nypl-colors-ui-bg-default);
+  border-color: var(--nypl-colors-ui-gray-semi-dark) !important;
+  border-left: 4px solid;
+  font-size: var(--nypl-fontSizes-desktop-caption);
+  font-weight: 510;
+  font-style: normal !important;
+}
+
+.statusBadgeReady {
+  background: rgba(238, 247, 240, 1) !important;
+  border-color: var(--nypl-colors-ui-success-primary) !important;
+  border-left: 4px solid;
+  font-size: var(--nypl-fontSizes-desktop-caption);
+  font-weight: 510;
```

Comment:
"510" is used a lot in this PR. The closest DS design token value for this is "medium" which makes to "500" -- in this case `font-weight: var(--nypl-fontWeights-medium);`. Likewise, in DS components you can pass `fontWeight: "medium"`.

Use the design token value to align with the DS value.

Reply:
sorry this didnt get communicated, but this styling is not going to be necessary once we incorporate the v3 update that includes the new status badges. So don't worry about changing this!